RFCs/0097-declarativeCDK

```diff
@@ -0,0 +1,48 @@
+# Declarative CDK
+
+* **Original Author(s):**: @evgenyka
+* **Tracking Issue**: #97
+
+# Summary
+
+Introducing Declarative CDK - a project that allows non-developer customers use CDK Constructs in their CloudFormation templates. 
```

Comment:
Maybe non-developers are the target audience, but no need to mention them here. *Anyone* will be able to use CDK Constructs in CFN templates

Reply:
Also: people writing yaml _are_ developers

Reply:
Updated.